MonoGame.Framework/BoundingFrustum.cs

```diff
@@ -123,7 +123,7 @@ internal string DebugDisplayString
         #region Constructors
 
         /// <summary>
-        /// Creates a new instance of <see cref="BoundingFrustum"/> class.
+        /// Constructs a bounding frustum.

```

Comment:
So this may sound like i'm picking on you @Shqrdx or I'm just being mean...  that is not my intention.  I'm just *very* passionate about getting good docs into MonoGame.

This is an example of bad documentation.  It does nothing but repeat the obvious...  you know it is a constructor... you know it is a bounding frustum.  Yes in some classes there is not more to say...  as the constructor does nothing useful.. but not in this case.  

Let's actually explain what the constructor is doing:
```
/// Constructs the frustum by extracting the view planes from a matrix.
```
This actually provides information that isn't included in the function signature and will inform new users as to exactly how this class works.